api/server/v1/marshaler.go

```diff
@@ -732,6 +732,97 @@ func (x *UpdateNamespaceMetadataRequest) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// UnmarshalJSON for SetRequest.
+func (x *SetRequest) UnmarshalJSON(data []byte) error {
```

Comment:
Can we move cache-related stuff in `cache.go`?

Reply:
I didn't realize we follow this pattern. Moved it to cache.go Thanks!.